### PR TITLE
Add instructions to set up dhcp server

### PR DESCRIPTION
**Pull request by TerryHowe**
Add some basic instructions on DIY DHCP server for vSphere

### CODE REVIEW

File: docs/content/en/docs/reference/vsphere/vsphere-dhcp.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+range 10.8.105.9  10.8.105.41;
+option subnet-mask 255.255.255.0;
+option routers 10.8.105.1;
+ option domain-name-servers 147.149.1.69;
+}
+```
+ 
+# Configure /etc/default/isc-dhcp-server
+
+Add the main interface to this file e.g: eth0 (this example uses ens160).
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/vsphere/vsphere-dhcp.md:41]:
```suggestion
Add the main NIC device interface to this file, such as eth0 (this example uses ens160).
```

File: docs/content/en/docs/reference/vsphere/vsphere-dhcp.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+ 
+```
+INTERFACESv4="ens160"
+```
+ 
+# Restart DCHP
+
+```
+service isc-dhcp-server restart
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/vsphere/vsphere-dhcp.md:50]: Is the service enabled by default? With Fedora, I think a separate step would be required to enable. If that is all that's needed in Ubuntu, ignore this comment.

**Review by chrisnegus — COMMENTED** [review]
I have a few suggestions. With those corrections, I think it's ready to go.

File: docs/content/en/docs/reference/vsphere/vsphere-dhcp.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+---
+title: "Custom DHCP Configuration"
+weight: 40
+description: >
+  Create a custom DHCP configuration for your vSphere deployment
+---
+
+If your vSphere deployment configred with DHCP and it cannot be configured with DHCP, you may want to run your own DHCP server.
+It may be necessary to turn off DHCP snooping on your switch to get DHCP working across VM main-servers.
```
[the remark this answers is not in the document]
Comment (TerryHowe) [reply docs/content/en/docs/reference/vsphere/vsphere-dhcp.md:9]: I'm not entirely sure either. I'll just change that to servers.

File: docs/content/en/docs/reference/vsphere/vsphere-dhcp.md (hunk 2 above)
Comment (TerryHowe) [reply docs/content/en/docs/reference/vsphere/vsphere-dhcp.md:50]: This looks correct to me. Installation seems to enable the service.

**Pull request merged**